cli/azd/cmd/auth_login.go

```diff
@@ -118,6 +119,14 @@ func (lf *loginFlags) Bind(local *pflag.FlagSet, global *internal.GlobalCommandO
 	)
```

Comment:
"When true, log in by using a device code instead of a browser." -- this comment feels outdated